docs/layers/scenegraph-layer.md

```diff
@@ -0,0 +1,159 @@
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/@deck.gl/mesh--layers-lightgrey.svg?style=flat-square" alt="@deck.gl/mesh-layers" />
+  <img src="https://img.shields.io/badge/lighting-TBD-blue.svg?style=flat-square" alt="lighting" />
+</p>
+
+# ScenegraphLayer
+
+The `ScenegraphLayer` renders a number of instances of a complete luma.gl scenegraph. While scenegraphs can be created programmatically, they are typically obtained by loading glTF files.
+
+```js
+import DeckGL from '@deck.gl/react';
+import {ScenegraphLayer} from '@deck.gl/mesh-layers';
+
+const App = ({data, viewport}) => {
+
+  /**
+   * Data format:
+   * [
+   *   {
+   *     position: [-122.45, 37.7],
+   *     angle: 0,
+   *     color: [255, 0, 0]
+   *   },
+   *   {
+   *     position: [-122.46, 37.73],
+   *     angle: 90,
+   *     color: [0, 255, 0]
+   *   },
+   *   ...
+   * ]
+   */
+  const layer = new ScenegraphLayer({
+    id: 'mesh-layer',
+    data,
+    scenegraph: 'scenegraph.gltf'
+  });
+
+  return (<DeckGL {...viewport} layers={[layer]} />);
+};
+```
+
+## Installation
+
+To install the dependencies from NPM:
+
+```bash
+npm install deck.gl
+# or
+npm install @deck.gl/core @deck.gl/mesh-layers
+```
+
+```js
+import {ScenegraphLayer} from '@deck.gl/mesh-layers';
+new ScenegraphLayer({});
+```
+
+To use pre-bundled scripts:
+
+```html
+<script src="https://unpkg.com/@deck.gl@~7.0.0/dist.min.js"></script>
+<!-- or -->
+<script src="https://unpkg.com/@deck.gl/core@~7.0.0/dist.min.js"></script>
+<script src="https://unpkg.com/@deck.gl/mesh-layers@~7.0.0/dist.min.js"></script>
+```
+
+```js
+new deck.ScenegraphLayer({});
+```
+
+
+## Properties
+
+Inherits from all [Base Layer](/docs/api-reference/layer.md) properties.
+
+
+### Mesh
+
+##### `scenegraph` (URL|Object|Promise)
```

Comment:
@ibgreen Is it a plan to use loaders.gl core to load async props? As it stands any URL will be loaded using `fetch` and parsed as JSON.

Reply:
Yeah, I'm removing this now.

Reply:
Is there any way for async props to trigger loaders gl?

Reply:
We'll need to change how async props are implemented. I recommend doing this in a separate PR.